xbmc/threads/Thread.cpp

```diff
@@ -20,6 +20,7 @@
  *
  */
 
+#include "Application.h"
```

Comment:
anyway, including Application.h here is no-go and ruins the original intent of this code here. It does not have a dependency to Log.h either for this reason.